docs/index.md

```diff
@@ -5,8 +5,8 @@
 
 # BandBook
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S2-CS2103T-T15-3/tp/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://app.codecov.io/gh/AY2324S2-CS2103T-T15-3/tp)
+[![CI Status](https://github.com/AY2324S2-CS2103T-T15-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![codecov](https://codecov.io/gh/AY2324S2-CS2103T-T15-3/tp/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```

Comment:
Won't these links lead to the ones for AB3 instead of the ones for our own project?